evaluator/builtin_time.go

```diff
@@ -390,6 +390,16 @@ func builtinCurrentTime(args []types.Datum, _ context.Context) (d types.Datum, e
 	return convertToDuration(d, fsp)
 }
 
+// See https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_utc-date
+func builtinUTCDate(args []types.Datum, _ context.Context) (d types.Datum, err error) {
+	year, month, day := time.Now().Date()
+	t := mysql.Time{
+		Time: time.Date(year, month, day, 0, 0, 0, 0, time.Local),
```

Comment:
Date/Datetime/timestamp use the same data structure mysql/time.go. So when it store a date value, the hour/minute/second are all zero. We can not drop them before.
tidb> select cast(current_date() as SIGNED);
+--------------------------------+
| cast(current_date() as SIGNED) |
+--------------------------------+
| 20160402000000                 |
+--------------------------------+
1 row in set (0.01 sec)

mysql> select cast(current_date() as SIGNED);
+--------------------------------+
| cast(current_date() as SIGNED) |
+--------------------------------+
|                       20160402 |
+--------------------------------+
1 row in set (0.00 sec)

It is OK to raise a new issue and fix this bug in another PR. We can focus on UTC time in this PR.